airflow/models/xcom.py

```diff
@@ -785,19 +801,26 @@ def __getstate__(self) -> Any:
             return (str(statement), query.count())
 
     def __setstate__(self, state: Any) -> None:
+        """Construct a query by retrieving the statement from the `state` and assign it to the `self._query`class property."""
         statement, self._len = state
         self._query = Query(XCom.value).from_statement(text(statement))
 
     def __len__(self):
+        """Declare a `self._len` class property that assigns the number of rows the query returns using `query.count()` if the `self._len` property is `None`.
+        
+        Otherwise return `self._len`
+        """
         if self._len is None:
             with self._get_bound_query() as query:
                 self._len = query.count()
         return self._len
 
     def __iter__(self):
+        """Return a `_LazyXComAccessIterator` class instance to manage query sessions."""
```

Comment:
Better replace by Iterate over the ..., and try to avoid mention private/internal classes because them not part of the public API

Reply:
Noted. I'll make the changes.